src/peft/tuners/lora.py

```diff
@@ -117,7 +118,100 @@ def __post_init__(self):
         self.peft_type = PeftType.LORA
 
 
-class LoraModel(torch.nn.Module):
+# This class has just been moved above
```

Comment:
I don't think this type of comment is necessary. I just meant it should be included in the description of the PR for the reviewers to make their lives easier :)

Also, the comment is not quite true because `LoraLayer` has been changed a little bit :D

Reply:
makes sense :D I thought you meant to explicitly add comments about that, removed it!